lib/Cluster.js

```diff
@@ -463,8 +492,20 @@ class Cluster extends EventEmitter {
       req.push(config);
     }
     if (req.length) {
-      debug(this.constructor.NAME, 'configure reporting', req);
-      const { reports } = await super.configureReporting({ reports: req });
+      // Check if command should get manufacturerSpecific flag
+      const manufacturerId = this._checkForManufacturerSpecificAttributes(
+        Object.keys(attributes).map(n => {
+          const attr = this.constructor.attributes[n];
+          return attr.id;
```

Comment:
nit: maybe this is cleaner as `return this.constructor.attributes[n].id`
same at `writeAttributes`

Reply:
Agreed!